packages/decorators/src/djs-decorators.ts

```diff
@@ -4,8 +4,10 @@ import { Message, PermissionResolvable, Permissions } from 'discord.js';
 import { createFunctionPrecondition, FunctionFallback } from './utils';
 
 export const enum DecoratorIdentifiers {
-	RequiresPermissionsGuildOnly = 'requiresPermissionsGuildOnly',
-	RequiresPermissionsMissingPermissions = 'requiresPermissionsMissingPermissions'
+	RequiresBotPermissionsGuildOnly = 'requiresBotPermissionsGuildOnly',
+	RequiresBotPermissionsMissingPermissions = 'requiresBotPermissionsMissingPermissions',
```

Comment:
Should use Client instead of Bot to match with /framework's option (`requiredClientPermissions`) as well as the fact that bots are referred to as client.
```suggestion
	RequiresClientPermissionsGuildOnly = 'requiresClientPermissionsGuildOnly',
	RequiresClientPermissionsMissingPermissions = 'requiresClientPermissionsMissingPermissions',
```

Reply:
Of course, apply a rename to update all references ^w^